src/post.njk

```diff
@@ -39,3 +39,14 @@ permalink: "views/{{ postItem.slug }}/"
 	{% endif %}
 </article>
 {% endblock %}
+
+{% block footerScripts %}
+<script src="/js/content-header-observer.js"></script>
+<script>
+	// Track all h1 and h2 in content for the Views post page.
+document.querySelectorAll("main article.post-article h1, main article.post-article h2, header").forEach((section) => {
```

Comment:
Same as above: Why is `header` elements tracked?

Reply:
So when the header is in the top 6% of the viewport the side menu will stop highlighting items. See line `17` of `content-header-observer.js`